lib/helpers.js

```diff
@@ -26,6 +26,17 @@ _beforeEach.withApp = function(app) {
   });
 }
 
+_beforeEach.cleanDatasource = function(dsName) {
+  beforeEach(function() {
+    if(!dsName) dsName = 'db';
+
+    if (this.app) {
+      this.app.datasources[dsName].automigrate();
```

Comment:
@ritch thanks for following up. I forgot about that todo.

So your saying like this?

```js
_beforeEach.cleanDatasource = function(dsName, cb) {
  beforeEach(function() {
    //...
    cb();
```